### PR TITLE
Notes on upgrading to python3, and README updates.

### PR DESCRIPTION
**Pull request by richvdh**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -146,20 +143,21 @@ Installing prerequisites on OpenBSD::
 
 To install the Synapse homeserver run::
 
-    virtualenv -p python2.7 ~/.synapse
-    source ~/.synapse/bin/activate
+    mkdir -p ~/synapse
+    virtualenv -p python3 ~/synapse/env
```
Comment (erikjohnston) [inline README.rst]: Does python3 recommend using the venv module?

**Comment by richvdh** [comment]
> What's the rationale on changing the virtualenv directories?

It always struck me that putting them in a hidden directory was odd, and I don't know why we recommend it. I thought I'd clean it up while we were here.

File: README.rst (hunk 1 above)
Comment (richvdh) [reply README.rst]: I'm unfamiliar with the venv module.

Comment (erikjohnston) [reply README.rst]: Its a standard library equivalent to virtualenv, but probably fine to stick with `virtualenv` for now

**Comment by erikjohnston** [comment]
> > What's the rationale on changing the virtualenv directories?
> 
> It always struck me that putting them in a hidden directory was odd, and I don't know why we recommend it. I thought I'd clean it up while we were here.

Its a bit rude to be adding bits directly to the home directory, but eh, I guess if people care then they can change it when following the instructions

**Pull request merged**